### ISSUE TITLE
`'BROWSER' is not recognized as an internal or external command`

### ISSUE THREAD

**Issue by littleclown** [yhirose/react-typescript-electron-sample-with-create-react-app-and-electron-builder#16]
The original scripts will get warning in powershell and exit code 1.

I found a solution maybe to fix this problem:

Use the following config
```
"scripts": {
  "electron:dev": "concurrently \"cross-env BROWSER=none yarn start\" \"wait-on http://localhost:3000 && tsc -p electron -w\" \"wait-on http://localhost:3000 && tsc -p electron && electron .\"",
},
"devDependencies": {
  "cross-env": "^7.0.3"
}
```

instead of 
```
"scripts": {
  "electron:dev": "concurrently \"BROWSER=none yarn start\" \"wait-on http://localhost:3000 && tsc -p electron -w\" \"wait-on http://localhost:3000 && tsc -p electron && electron .\"",
}
```

in `package.json`

**Comment by yhirose** [comment]
@littleclown, thanks for the feedback. Could you send a pull request for the change?

**Comment by yhirose** [comment]
Fixed wt #25.
